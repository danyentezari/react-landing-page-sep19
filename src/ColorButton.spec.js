import React from 'react';
import ColorButton from './ColorButton';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
Enzyme.configure({ adapter: new Adapter() });

describe('ColorButton', ()=>{

    let wrapper;

    beforeEach(()=>{
        wrapper = shallow(
            <ColorButton />
        )
    });

    it('renders without crashing', ()=> {
        expect(wrapper).toMatchSnapshot();
    });

    it('has btn-success class when clicked', () => {
        const btn = wrapper.find('.btn');
        btn.simulate('click');
        expect(wrapper.find('.btn-success')).toHaveLength(1)
    });

    it('will turn red when hovered')
})