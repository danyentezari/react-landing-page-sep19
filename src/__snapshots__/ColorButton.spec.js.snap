// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ColorButton renders without crashing 1`] = `ShallowWrapper {}`;
